src/platform.ts

```diff
@@ -128,7 +129,18 @@ export class DreoPlatform implements DynamicPlatformPlugin {
 
         // create the accessory handler for the restored accessory
         // this is imported from `platformAccessory.ts`
-        new FanAccessory(this, existingAccessory, state, ws);
+        switch (device.seriesName) {
+          case 'fan':
```

Comment:
Does Dreo actually use the name 'fan' for any devices in their API? Mine is named 'Tower Fan' which causes the plugin to skip over it and print 'error, unknown device type'